test/Array/array_indexOf.baseline

```diff
@@ -664,6 +664,7 @@
 -1
 -1
 -1
+5

```

Comment:
I written a post it to keep it in mind. I'll do it late June if I have the time.

---
In reply to: [115389276](https://github.com/Microsoft/ChakraCore/pull/2898#discussion_r115389276) [](ancestors = 115389276)